libs/designsystem/src/lib/components/section-header/section-header.component.scss

```diff
@@ -0,0 +1,17 @@
+@use '~@kirbydesign/core/src/scss/utils';
+@use './section-header.utils';
+
+:host {
+  ion-item-divider {
+    --inner-padding-end: 0;
+    --color: unset;
+    --background: transparent;
+    font-weight: inherit;
+    // TODO: Determine if we really want this.
+    // This way, section header cannot be used as simple spacer
+    // Without supplying a piece of content with the needed height
+    min-height: 0;
+
+    @include section-header.typography();
```

Comment:
Consider moving this up _before_ other "normal" declarations, but _after_ custom property declarations.